modules/simple_forms_api/app/services/simple_forms_api/pdf_stamper.rb

```diff
@@ -9,15 +9,11 @@ class PdfStamper
     SUBMISSION_DATE_TITLE = 'Application Submitted:'
 
     def self.stamp_pdf(stamped_template_path, form, current_loa)
-      if File.exist? stamped_template_path
-        stamp_signature(stamped_template_path, form)
+      stamp_signature(stamped_template_path, form)
```

Comment:
I'm removing this check on `stamped_template_path` because it is always preceded by this identical check: https://github.com/department-of-veterans-affairs/vets-api/blob/f2c7df6191b3d2223b232fc65823171725f13d9a/modules/simple_forms_api/app/services/simple_forms_api/pdf_filler.rb#L26